ci/test-clang.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -eu
+
+trap 'rm -fr "$toolchain"' EXIT
+
+# Set up workspace
+rootdir=${GITHUB_WORKSPACE:-$(dirname "$(dirname "$(realpath "$0")")")}
+toolchain=$rootdir/toolchain
+
+# Pull toolchain out of container
+echo "[+] Downloading toolchain from container"
+docker create --name llvm-project ghcr.io/clangbuiltlinux/llvm-project:stage2
+mkdir -p "$toolchain"/{bin,include,lib}
```

Comment:
will `docker cp` fail without this?

Reply:
I think so, I'll double check though!

Reply:
I've make this a little more traditional with b8a02d204216d65c3c86d8e0774828ac2e2e2752, let me know if that is better or worse to you.